pkg/ebpf/common.go

```diff
@@ -89,6 +89,10 @@ func verifyOSVersion(kernelCode uint32, platform string, exclusionList []string)
 		if kernelCode >= linuxKernelVersionCode(4, 4, 119) && kernelCode <= linuxKernelVersionCode(4, 4, 126) {
 			return false, fmt.Sprintf("got ubuntu kernel %s with known bug on platform: %s, see: https://bugs.launchpad.net/ubuntu/+source/linux/+bug/1763454", kernelCodeToString(kernelCode), platform)
 		}
+	} else if isLinuxAWS(platform) {
+		if kernelCode < linuxKernelVersionCode(4, 4, 128) {
```

Comment:
How do you get 128 from `kernel 4.4.0-1060.69` ?

Reply:
When we call [`CurrentKernelVersion()`](https://github.com/iovisor/gobpf/blob/362e99fa21988d6b1eecbb3aa683e55a4a622029/elf/kernel_version.go#L103) it ends up returning the release string at the end of the version signature for Ubuntu. So when I ran this in an EC2 it resolved to `4.4.128`, with `128` being the kernel patch number:

```
ubuntu@ip-###-##-###-##:~$ cat /proc/version_signature
Ubuntu 4.4.0-1060.69-aws 4.4.128
```